src/enrollments/EnrollmentCollectionEntity.js

```diff
@@ -16,4 +16,12 @@ export class EnrollmentCollectionEntity extends Entity {
 		const enrollmentEntities = this._entity.getSubEntitiesByRel(Rels.userEnrollment);
 		return enrollmentEntities.map(e => e.href).filter(href => href);
 	}
+
+	getActionByName(url) {
```

Comment:
I am using an argument here because in https://github.com/Brightspace/d2l-my-courses-ui/blob/master/src/d2l-my-courses-behavior.js#L84 `getActionByName` takes `Actions.enrollments.searchMyEnrollments` or `Actions.enrollments.searchMyPinnedEnrollments`

Reply:
Better to have `searchMyEnrollmentsAction` and `searchMyPinnedEnrollmentsAction` in the SDK, so we can remove `Actions` from my courses later

Reply:
Why did we get away from this the above?

Reply:
@m6jones I am not sure I understand what you mean?

Reply:
Yeah I agree, I think what we eventually want is that `d2l-hypermedia-constants` will mostly go away.  Each `Entity` will have the strings of the actions that make sense to it, and this will be the only place they exist (consumers like my-courses don't need to know the action strings, they just call a function).  That way, if the API or action name ever changed, we'd update the SDK and all the other components get it for free